src/scrapers/main.py

```diff
@@ -37,9 +40,16 @@ def docs():
     docs()
 else:
     print(f'Running {len(args)} scrapers. Use --help to see all individual scrapers')
+    if opts.group:
+        if opts.group != InitiativeGroup.SUPPLY \
+                and opts.group != InitiativeGroup.DEMAND:
+            print(f"Invalid group given {opts.group}")
+            exit()
     for scraper in scrapers:
         if opts.limit:
             scraper.limit = opts.limit
+        if opts.group:
+            scraper.set_group(InitiativeGroup.DEMAND if opts.group == "demand" else InitiativeGroup.SUPPLY)
```

Comment:
Als de parser goed werkt dan kun je hier gewoon opts.group in plempen toch?

Reply:
Heb type restriction of die setter gezet. Dan moet het volgens mij wel zo?